View/CsvView.php

```diff
@@ -293,8 +293,15 @@ protected function _generateRow($row = null) {
 		$newline = $this->viewVars['_newline'];
 
 		$row = str_replace(array("\r\n", "\n", "\r"), $newline, $row);
-		if (fputcsv($fp, $row, $delimiter, $enclosure) === false) {
-			return false;
+		if ($enclosure == '') {
```

Comment:
Strict checking to null seems to be a bit more problematic.

In the _setupViewVars function all CSV related view variables checked for being set and if they are not set then they initialized as null. Then the variables with null value are set to the proper default value:
https://github.com/FriendsOfCake/cakephp-csvview/blob/master/View/CsvView.php#L153

We can do strict checking at the View, but that would require some work on the _setupViewVars part too, which is not a problem for me, but I think it would break the current flow there.  So I would vote for the checking against the empty string ('') in the view. 

Let me know your opinion! Thanks!